src/frontend-config/frontend-config.dto.ts

```diff
@@ -16,68 +16,67 @@ import { URL } from 'url';
 
 import { ServerVersion } from '../monitoring/server-status.dto';
 
-export class AuthProviders {
-  /**
-   * Is Facebook available as a auth provider?
-   */
-  @IsBoolean()
-  facebook: boolean;
-
-  /**
-   * Is GitHub available as a auth provider?
-   */
-  @IsBoolean()
-  github: boolean;
-
-  /**
-   * Is Twitter available as a auth provider?
-   */
-  @IsBoolean()
-  twitter: boolean;
+export enum AuthProviderType {
+  INTERNAL = 'internal',
+  LDAP = 'ldap',
+  SAML = 'saml',
+  OAUTH2 = 'oauth2',
+  GITLAB = 'gitlab',
+  FACEBOOK = 'facebook',
+  GITHUB = 'github',
+  TWITTER = 'twitter',
+  DROPBOX = 'dropbox',
+  GOOGLE = 'google',
+}
 
-  /**
-   * Is at least one GitLab server available as a auth provider?
-   */
-  @IsBoolean()
-  gitlab: boolean;
+export type AuthProviderTypeWithCustomName =
+  | AuthProviderType.LDAP
+  | AuthProviderType.OAUTH2
+  | AuthProviderType.SAML
+  | AuthProviderType.GITLAB;
 
-  /**
-   * Is DropBox available as a auth provider?
-   */
-  @IsBoolean()
-  dropbox: boolean;
+export type AuthProviderTypeWithoutCustomName =
+  | AuthProviderType.INTERNAL
+  | AuthProviderType.FACEBOOK
+  | AuthProviderType.GITHUB
+  | AuthProviderType.TWITTER
+  | AuthProviderType.DROPBOX
+  | AuthProviderType.GOOGLE;
 
+export class AuthProviderWithoutCustomName {
```

Comment:
These classes should have `Dto` at the end of their name

Reply:
Fixed